app/views/two_factor_authentication/sms_opt_in/error.html.erb

```diff
@@ -26,6 +26,7 @@
         decorated_session.sp_name && {
           url: return_to_sp_cancel_path(location: 'sms_resubscribe_error'),
           text: t('idv.troubleshooting.options.get_help_at_sp', sp_name: decorated_session.sp_name),
+          new_tab: true,
```

Comment:
Actually, should this be on the second link instead? (The one to the marketing site link)

Reply:
ah whoops, I misread the comment on the FigJam, moved it in bb95f4c0363c56d7853809c7dd39fec8bf6758cb